[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-1 (#114270)
openshift/openshift-docs: modules/oc-compliance-fetching-raw-results.adoc

Reviewer: The format of this proc is not rendering as a list of steps - missing indent and related number restart

<img width="375" height="881" alt="Image" src="https://github.com/user-attachments/assets/ec53fc6a-5953-4bde-aa97-8652b2c3c66b" />

[id="fetching-raw-results_{context}"]
= Fetching raw results

[role="_abstract"]

> When a compliance scan finishes, the results of the individual checks are listed in the resulting `ComplianceCheckResult` custom resource (CR). However, an administrator or auditor might require the complete details of the scan. The OpenSCAP tool creates an Advanced Recording Format (ARF) formatted file with the detailed results. This ARF file is too large to store in a config map or other standard Kubernetes resource, so a persistent volume (PV) is created to contain it.

Reviewer 2: Length

```suggestion
An administrator or auditor can review the complete detailed results of a scan as created by the OpenSCAP tool. These results contain more details than what is contained in the  `ComplianceCheckResult` custom resource (CR).

When a compliance scan finishes, the results of the individual checks are listed in the resulting `ComplianceCheckResult` custom resource (CR). The OpenSCAP tool creates an Advanced Recording Format (ARF) formatted file with the complete detailed results of a scan. This ARF file is too large to store in a config map or other standard Kubernetes resource, so a persistent volume (PV) is created to contain it.
```

.Procedure

> . Fetch the results from the PV with the Compliance Operator is a four-step process. However, with the `oc-compliance` plugin, you can use a single command:

Reviewer 2: ```suggestion
. Fetch the results from the PV by running the following command:
```

+
[source,terminal]
----
$ oc compliance fetch-raw <object-type> <object-name> -o <output-path>
----
+
where:

`<object-type>`:: It can be either `scansettingbinding`, `compliancescan` or `compliancesuite`, depending on which of these objects the scans were launched with.

> `<object-name>`:: Specifies the name of the binding, suite, or scan object to gather the ARF file for, and `<output-path>` is the local directory to place the results.

Reviewer 2: I think we can leave the original text. Avoids having to rewrite the awkward sentence for object-type.

Author: reverted to original
